### PR TITLE
Add random brightness to Image Preprocessing (Code Cleanup)

### PR DESCRIPTION
**Pull request by abrad1212**
**This is a cleanup of @zhudaoruyi's PR at #8652.**
**All credit goes to him!**

### CODE REVIEW

**Comment by abrad1212** [comment]
Also I used **AutoPep8** to format the code :)

File: keras/preprocessing/image.py [hunk 1]
```diff
@@ -382,6 +391,7 @@ class ImageDataGenerator(object):
         featurewise_std_normalization: divide inputs by std of the dataset.
         samplewise_std_normalization: divide each input by its std.
         zca_whitening: apply ZCA whitening.
+        brightness: apply brightness
```
Comment (fchollet) [inline keras/preprocessing/image.py]: What does this mean?

File: keras/preprocessing/image.py [hunk 2]
```diff
@@ -684,6 +708,11 @@ def random_transform(self, x, seed=None):
             if np.random.random() < 0.5:
                 x = flip_axis(x, img_row_axis)
 
+        if self.brightness:
+            x = random_brightness(
+                x, np.random.uniform(
+                    0.1, self.brightness_range))
```
Comment (Dref360) [inline keras/preprocessing/image.py]: Can't you lower brightness as well?

File: keras/preprocessing/image.py (hunk 1 above)
Comment (abrad1212) [reply keras/preprocessing/image.py]: @fchollet I guess I wasn't paying attention. I meant for it to mean, Use Random Brightness

File: keras/preprocessing/image.py (hunk 2 above)
Comment (abrad1212) [reply keras/preprocessing/image.py]: @Dref360 I don't follow

File: keras/preprocessing/image.py (hunk 1 above)
Comment (fchollet) [reply keras/preprocessing/image.py]: That's redundant with `brightness_range`. We use only one argument for such transforms.

**Comment by abrad1212** [comment]
@fchollet I believe this is ready.

File: keras/preprocessing/image.py [hunk 3]
```diff
@@ -429,6 +444,7 @@ def __init__(self,
                  rotation_range=0.,
                  width_shift_range=0.,
                  height_shift_range=0.,
+                 brightness_range=0.,
```
Comment (Dref360) [inline keras/preprocessing/image.py]: This is not a range

Comment (abrad1212) [reply keras/preprocessing/image.py]: @Dref360 I was just following the others. Will this still work? You just have to provide a tuple of floats.

Comment (Dref360) [reply keras/preprocessing/image.py]: I would put None in this case. I know Python is not typed, but if the default is 0 and we expect a tuple, it's confusing

Comment (abrad1212) [reply keras/preprocessing/image.py]: Alright

**Pull request merged**